frame/support/src/traits/misc.rs

```diff
@@ -380,7 +380,7 @@ impl<T: Default> Get<T> for GetDefault {
 
 macro_rules! impl_const_get {
 	($name:ident, $t:ty) => {
-		#[derive($crate::RuntimeDebug)]
+		#[derive($crate::RuntimeDebug, Clone)]
```

Comment:
Can't we just define this ourselves instead of always having it implicitly defined? Worried about unnecessary overhead if we haven't needed it until now

Reply:
de80c7be8822669181e9a2a7c15d9c7117d4d28e